ibis/pandas/execution/tests/test_timecontext.py

```diff
@@ -0,0 +1,54 @@
+import pandas as pd
+import pandas.util.testing as tm
+import pytest
+
+import ibis
+
+pytestmark = pytest.mark.pandas
+
+
+def test_execute_with_timecontext(time_table):
+    expr = time_table
+    # define a time context for time-series data
+    context = ("20170101", "20170103")
```

Comment:
In the doc for context type, you used `Tuple[pd.Timestamp, pd.Timestamp]`, seems that you would want to define context as `(pd.Timestamp("20170101"), pd.Timestamp("20170103"))`?

Reply:
thx, yes